scripts/util.py

```diff
@@ -309,6 +311,57 @@ def blob_list(prefix="", delimiter=None, project=None):
     )
     return [blob for blob in blobs]
 
+def install_custom_scripts(check_hash=False):
+    """download custom scripts from gcs bucket"""
+
+    compute_tokens = ["compute", "prolog", "epilog"]
+    if lkp.instance_role == "compute":
+        try:
+            compute_tokens.append(f"nodeset-{lkp.node_nodeset_name()}")
+        except Exception as e:
+            log.error(f"Failed to lookup nodeset: {e}")
+
+    prefix_tokens = dict.get(
+        {
+            "login": ["login"],
+            "compute": compute_tokens,
+            "controller": ["controller", "prolog", "epilog"],
+        },
+        lkp.instance_role,
+        [],
+    )
+    prefixes = [f"slurm-{tok}-script" for tok in prefix_tokens]
+    blobs = list(chain.from_iterable(blob_list(prefix=p) for p in prefixes))
```

Comment:
nit. no need to convert to `list`

Reply:
This part of the code is jsut moved from setup.py to util.py, did not review it.